Content.Shared/Maps/TurfHelpers.cs

```diff
@@ -120,13 +123,40 @@ public static bool TryGetTileRef(this EntityCoordinates coordinates, [NotNullWhe
         /// <summary>
         ///     Helper that returns all entities in a turf.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static IEnumerable<IEntity> GetEntitiesInTile(this TileRef turf, bool approximate = false)
         {
             var entityManager = IoCManager.Resolve<IEntityManager>();
```

Comment:
This can be made a nullable parameter as well.

Reply:
done